src/slackbot/checkly.spec.ts

```diff
@@ -0,0 +1,251 @@
+import { expect } from "@jest/globals";
+import { WebClient } from "@slack/web-api";
+import dotenv from "dotenv";
+import {
+  summarizeErrorsPrompt,
+  SummarizeErrorsPromptType,
+} from "../prompts/checkly";
+import { generateObject } from "ai";
+import { ChecklyClient } from "../checkly/checklyclient";
+import {
+  fetchCheckResults,
+  summarizeCheckResult,
+  LAST_30_DAYS,
+  last24h,
+} from "../prompts/checkly-data";
+import { CheckResult } from "../checkly/models";
+import { createHeatmap, generateHeatmapPNG } from "../heatmap/createHeatmap";
+import { analyzeImageBufferForPatterns } from "../heatmap/analyseHeatmap";
+
+dotenv.config();
+
+describe("Checkly Slack Message Tests", () => {
+  const checkly = new ChecklyClient({});
+  const slackClient = new WebClient(process.env.SLACK_AUTH_TOKEN!);
+  const channel = process.env.SLACK_BOT_CHANNEL_ID!;
+
+  it.skip("Summarize a single Check Result and send a Slack Notification", async () => {
```

Comment:
It's skipped because it's the demo test?

Reply:
this is absically stuff which should be in a notebook, will be cleaned up with the slash command